DTAdmin/Groups/Entities.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  Entities.swift
+//  DTAdmin
+//
+//  Created by Kravchuk on 18.10.17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import Foundation
+
+class Group {
+    var id: String
+    var name: String
+    var facultyId: String
+    var specialityId: String
+//    var facultyName: String?
```

Comment:
Please remove commented dead code or provide good justification why have you commented it.